webapp/src/app/components/dialogs/map-settings/map-content-settings/map-content-settings.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit, Input, EventEmitter, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-map-content-settings',
+  templateUrl: './map-content-settings.component.html',
+  styleUrls: ['./map-content-settings.component.scss', '../map-settings.component.scss']
```

Comment:
should not implement styles from other components. Because of this refactoring all styles from
`map-settings.component.scss` can be simply copied over to this one and delete the old styles. Same with the new map component